### PR TITLE
[UPDATE] Banner & new badges

### PR DESCRIPTION
**Pull request by ilyalehchylin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
-[![Build Status](https://app.bitrise.io/app/d80a106bb67fa6e8/status.svg?token=xnyYwDyHqw773UCliHgd4g&branch=develop)](https://app.bitrise.io/app/d80a106bb67fa6e8)
-[![Maintainability](https://api.codeclimate.com/v1/badges/0265f119ef5164d3ee70/maintainability)](https://codeclimate.com/github/bykovme/nswallet/maintainability)
+[![NS Wallet](https://github.com/bykovme/nswallet/blob/update/badges/banner.png?raw=true)](https://nswallet.com/)
 
-## NS Wallet 
+# NS Wallet 
 
+[![Framework](https://img.shields.io/badge/framework-Xamarin.Forms-3498DB?logo=xamarin&logoColor=white)](https://github.com/xamarin/Xamarin.Forms)
```
Comment (bykovme) [inline README.md:5]: Image is not shown

File: README.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
-[![Build Status](https://app.bitrise.io/app/d80a106bb67fa6e8/status.svg?token=xnyYwDyHqw773UCliHgd4g&branch=develop)](https://app.bitrise.io/app/d80a106bb67fa6e8)
-[![Maintainability](https://api.codeclimate.com/v1/badges/0265f119ef5164d3ee70/maintainability)](https://codeclimate.com/github/bykovme/nswallet/maintainability)
+[![NS Wallet](https://github.com/bykovme/nswallet/blob/update/badges/banner.png?raw=true)](https://nswallet.com/)
```
Comment (bykovme) [inline README.md:1]: Change description on the image to "Free Offline Multi-platform Password Manager"

**Review by bykovme — COMMENTED** [review]
fix readme, please

File: README.md (hunk 1 above)
Comment (ilyalehchylin) [reply README.md:5]: It's laggy today a little bit

File: README.md (hunk 2 above)
Comment (ilyalehchylin) [reply README.md:1]: Changed. Look at [commit 70287d7](https://github.com/bykovme/nswallet/commit/70287d7ba61d32103054e0e084d16e4ba4bb8a45).

**Pull request merged**